UI/data/locale.ini

```diff
@@ -65,10 +68,10 @@ Name=Svenska
 Name=Türkçe
 
 [zh-CN]
-Name=简体中文
+Name=中文简体
```

Comment:
I had this reviewed by native speakers, and the ones I asked agreed that the original text is better than the proposed change.